cdci_data_analysis/analysis/parameters.py

```diff
@@ -382,30 +393,34 @@ def value(self):
 
     @value.setter
     def value(self, v):
-        if v is not None and v!='':
+        if v is not None and v != '':
             self.check_float_value(v, name=self.name, units=self.units)
             self._v = np.float(v)
 
         else:
             self._v=None
 
+    def get_value_in_default_format(self, value):
+        return float(value)
+
     @staticmethod
     def check_float_value(value, units=None, name=None):
         #print('check type of ',name,'value', value, 'type',type(value))
-        if value is None or value=='':
+        if value is None or value == '':
             pass
         else:
             try:
-              value=ast.literal_eval(value)
+                value = ast.literal_eval(value)
```

Comment:
I do not know why is this except needed?

Reply:
in case an actual float value is passed

Reply:
I mean the whole block. It's either going to be converted with `float(value)` or it is not float-able. What could be the case which is string literal of float but not converted with `float`?

Reply:
I was indeed wondering the same, but in any case, it is gone now because non-necessary